core/src/main/java/com/scalar/db/storage/dynamo/SelectStatementHandler.java

```diff
@@ -109,15 +110,17 @@ private Scanner executeQueryWithIndex(Selection selection, TableMetadata tableMe
     ValueBinder binder = new ValueBinder(DynamoOperation.VALUE_ALIAS);
     keyColumn.accept(binder);
     Map<String, AttributeValue> bindMap = binder.build();
-    builder
-        .keyConditionExpression(condition)
-        .expressionAttributeValues(bindMap)
-        .expressionAttributeNames(ImmutableMap.of(expressionColumnName, column));
```

Comment:
This bug is introduced in #462.

We set `expressionAttributeNames` here, but we also set it in `projectionExpression()` if the operation has projections, so the `expressionAttributeNames` set here is overwritten, which is a problem.
https://github.com/scalar-labs/scalardb/pull/544/files#diff-59d53393b90fb6d234264b0143965e0670ab3a4e80341de6a4e79b3bf1d3c29aL177-R188